MultiServer.py

```diff
@@ -1337,6 +1337,7 @@ def _cmd_remaining(self) -> bool:
                     "Sorry, !remaining requires you to have beaten the game on this server")
                 return False
 
+    @mark_raw
     def _cmd_missing(self, filter_text="") -> bool:
```

Comment:
Not sure if this is the sort of thing PR authors want to hear about on this project, but I noticed https://github.com/ArchipelagoMW/Archipelago/pull/2538/files#diff-604f2185ff95ec6ead8869688d84dd861a5f701dfbe854335a019645546d222eR1340 makes this same change, so there might be a merge conflict coming soon there (cc @NewSoupVi ).